docs/conformance.md

```diff
@@ -102,6 +102,12 @@ To run without outside connectivity tests add the argument:
 SKIP_OUTSIDE_CONN_TESTS=true make conformance
 ```
 
+To run without block storage/snapshot tests add the arguments:
```

Comment:
nit: `SKIP_BLOCK_STORAGE_TESTS=true` is not an argument, but an environment variable.

I'd say explicitly that this make sense only if you do not have block storage and snapshot classes.

Reply:
thanks, done